azure-maven-plugin-lib/src/main/java/com/microsoft/azure/maven/AbstractAzureMojo.java

```diff
@@ -291,7 +291,10 @@ protected AuthType getAuthTypeEnum() {
                 .filter(authTypeEnum -> StringUtils.equalsAnyIgnoreCase(authTypeEnum.name(), authType))
                 .findFirst().orElse(null);
         if (result == null) {
-            Log.warn(String.format(INVALID_AUTH_TYPE, authType, StringUtils.join(AuthType.values(), ", ")));
```

Comment:
Please use simpler name for auth types other than strict enum values.

Reply:
Fixed in #981